packages/core/package.json

```diff
@@ -16,6 +16,9 @@
     "./typescript.svg",
     "./rest_hooks_logo_and_text.svg"
   ],
+  "engines": {
+    "node": ">=6.17.1"
```

Comment:
```suggestion
    "node": ">=6.5"
```
note that this also prevents IE 10, and somewhat IE 11, from being able to use rest-hooks, and a *lot* of people still have to support IE 11.

Reply:
https://caniuse.com/?search=JavaScript%20WeakMap what's the 'somewhat' ie11 about? ie10 isn't supported.

Reply:
http://kangax.github.io/compat-table/es6/ indicates that IE 11 has partial WeakMap support.

Reply:
(caniuse is not an effective choice for language things; the kangax table is the best source)

Reply:
node v4 is needed for some of the weakmap() features according to https://node.green/

Reply:
Looks like the only potential for IE11 is if people somehow try to inject frozen objects. That's not recommended anywhere in the docs so probably a fine tradeoff for them to consider.

Reply:
For iterator closing, sure, but you're not using that feature. It'd be node v6.5 if you needed that feature.